### PR TITLE
Implement binary encoding

### PR DESCRIPTION
**Pull request by stefnotch**
Takes care of a part of https://github.com/tuwien2020/tgi-pages/issues/14

Please do test this with a few inputs. It wouldn't surprise me if I overlooked something somewhere.

### CODE REVIEW

File: src/pages/BinaryCoding.vue [hunk 1]
```diff
@@ -1,19 +1,56 @@
 <template>
   <div>
     <h1>Bin&auml;re Codierungen</h1>
-    <!--<h2>Encoding</h2>
-    <p>Gebe hier eine Bin&auml;re Zahl ein:</p>-->
-    <!-- TODO: Encoding -->
+    <!-- TODO: Is this UI fine or is it confusing? -->
```
Comment (Fancy11111) [inline src/pages/BinaryCoding.vue:4]: i think the ui is fine

File: src/pages/BinaryCoding.vue [hunk 2]
```diff
@@ -69,25 +114,108 @@ export default defineComponent({
...
-      getBinaryNumber: (value: ReadonlyArray<boolean>, options: T) => BinaryNumber,
-      options: T
-    ) {
-      return {
-        name,
-        binaryNumber: computed(() => getBinaryNumber(bitPattern.value, options)),
-        options,
-      };
-    }
+    // TODO: Maybe we should strip the zeros at the beginning?
```
Comment (Fancy11111) [inline src/pages/BinaryCoding.vue:127]: do you still want to do this before merging or not? I don't think it is necessary

**Review by Fancy11111 — APPROVED** [review]
Looks good to me

File: src/pages/BinaryCoding.vue (hunk 1 above)
Comment (stefnotch) [reply src/pages/BinaryCoding.vue:4]: Sweet, thanks for the feedback

File: src/pages/BinaryCoding.vue (hunk 2 above)
Comment (stefnotch) [reply src/pages/BinaryCoding.vue:127]: Probably not, I'll go ahead and merge this

**Pull request merged**